R/model_parameters.merMod.R

```diff
@@ -43,13 +44,17 @@ model_parameters.merMod <- function(model, ci = .95, standardize = FALSE, bootst
 
   # p value
   if("Pr(>|z|)" %in% names(parameters)){
-    names(parameters) <- gsub("Pr(>|z|)", "p", names(parameters))
+    names(parameters)[grepl("Pr(>|z|)", names(parameters))] <- "p"
```

Comment:
Does this work without using `fixed = TRUE`? I think you else need to escape special regex chars.

Reply:
Yes, I would have to escape the characters for it to work, but I found it was quicker to use this alternative solution instead of struggling with the escape codes... 😅 
It is not less efficient, is it?

Reply:
I think it would be more safe to use `[grepl("Pr(>|z|)", names(parameters), fixed = TRUE)`, because `(` and `|` are special regex chars in your pattern-argument.

Reply:
You mean like this : `names(parameters)[grepl("Pr(>|z|)", names(parameters), fixed = TRUE)] <- "p"`?

Reply:
Yes!

Reply:
done :)